.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: node_js
+node_js:
+  - 4
+install:
+  - npm install bower -g
+  - npm install
+script:
+  - npm run build
+  - npm run build-example
+after_success:
+- >-
+  test $TRAVIS_TAG &&
+  ./node_modules/.bin/psc-publish > .pursuit.json &&
+  curl -X POST http://pursuit.purescript.org/packages \
+    -d @.pursuit.json \
+    -H 'Accept: application/json' \
+    -H "Authorization: token ${GITHUB_TOKEN}"
```

Comment:
This basically works verbatim for any pulp-built project, just need to tweak things if `npm run build-example` doesn't exist, or there's more than one example, etc.

You can see the `GITHUB_TOKEN` usage here also.